docs/io.cozy.accounts.md

```diff
@@ -143,31 +135,15 @@ in the `metadata` key of the relationship item.
 
 ```json
 {
+  "account_type": "freemobile",
   "auth": {
-    "accountName": "freemobile",
-    "credentials_encrypted": "aaDtwWSWsdpbbbbbbbbbbbbbbOzp2pBEbXlZLWjiTzOGumGRomrF2LwlRn4Y8c=",
-    "folderPath": "/Administratif/Free Mobile",
+    "credentials_encrypted": "aaDtwWSWsdpbbbbbbbbbbbbbbOzp...n4Y8c=",
     "login": "000000000",
-    "namePath": "Free Mobile",
-    "password": "*******"
-  }
-}
-```
-
-What we aim for:
-
-```json
-{
-  "auth": {
-    "credentials_encrypted": "aaDtwWSWsdpbbbbbbbbbbbbbbOzp2pBEbXlZLWjiTzOGumGRomrF2LwlRn4Y8c=",
-    "login": "000000000",
-    "password": "**********"
+    "password": "*******" // only when setting it first time
```

Comment:
JSON is a data-only format and doesn't support comments.
So this comment should not be inside json here (or if you choose to leave it, you can't use json type for this block code because it will fool the parser)